### DISCUSSION TITLE
How do I wrap agent A and agent B  as agent C, which C play as a part of the groupchat?

### DISCUSSION THREAD

**Discussion by HuXinjing** [microsoft/autogen#4370] [Q&A]
as title says, I find no examples could help me to do so.

**Comment by ekzhu** [comment]
@HuXinjing you need to create custom agent that wraps two agents. We don't have example for it at the moment. Perhaps you can dig into it a bit and give us an example? 

You can follow the agent chat tutorial to get started: https://microsoft.github.io/autogen/dev/user-guide/agentchat-user-guide/index.html

**Comment by richard-gyiko** [comment]
SocietyOfMindAgent maybe? It allows you to wrap a team into the agent.

https://microsoft.github.io/autogen/dev/reference/python/autogen_agentchat/autogen_agentchat.agents.html#autogen_agentchat.agents.SocietyOfMindAgent

**Reply by HuXinjing** [reply]
> SocietyOfMindAgent maybe? It allows you to wrap a team into the agent.
> 
> https://microsoft.github.io/autogen/dev/reference/python/autogen_agentchat/autogen_agentchat.agents.html#autogen_agentchat.agents.SocietyOfMindAgent

Great! It's what I'm looking for. Thx.

**Reply by realethanyang** [reply]
#5924 
AgentTool
TeamTool
